src/lib/rest/rest.cpp

```diff
@@ -922,14 +924,19 @@ static ApiVersion apiVersionGet(const char* path)
     return V2;
   }
 
-  // Different from v2, v1 is case-insensitive (see case/2057 test)
```

Comment:
I don't undrestand completly what to you mean with "add an enum to ConnectionInfo" but it is not important right now. I'll see (and review) when the moment comes.

NTC